linkis-commons/linkis-storage/src/main/java/org/apache/linkis/storage/LineRecord.java

```diff
@@ -15,22 +15,33 @@
  * limitations under the License.
  */
 
-package org.apache.linkis.storage.resultset.table
+package org.apache.linkis.storage;
 
-import org.apache.linkis.common.io.Record
-import org.apache.linkis.storage.resultset.ResultRecord
-import org.apache.linkis.storage.utils.StorageUtils
+import org.apache.linkis.common.io.Record;
+import org.apache.linkis.storage.resultset.ResultRecord;
 
-class TableRecord(val row: Array[Any]) extends ResultRecord {
+public class LineRecord implements ResultRecord {
+  public String line;
 
-  override def cloneRecord(): Record = {
-    new TableRecord(row)
+  public LineRecord(String line) {
+    this.line = line;
   }
 
-  def tableRecordToString(nullValue: String = "NULL"): Array[String] = {
-    row.map { col =>
-      StorageUtils.colToString(col, nullValue)
-    }
+  public String getLine() {
+    return line;
   }
 
+  public void setLine(String line) {
+    this.line = line;
+  }
+
+  @Override
+  public Record cloneRecord() {
+    return new LineRecord(line);
```

Comment:
if the line need be cloned also?

Reply:
Thanks, I double check if clone an Object need clone String field in website ,find the following result: because String object is immutable, therefore not clone String field also works fine. And I have  added a test for LineRecord cloneRecord method.